.gitignore

```diff
@@ -106,7 +106,11 @@ frameworks/CSharp/beetlex/Benchmarks/Properties/
 frameworks/CSharp/beetlex/Benchmarks/obj/
 frameworks/CSharp/beetlex/Benchmarks/.vs/
 frameworks/CSharp/beetlex/.vs/
+<<<<<<< HEAD
```

Comment:
You've left all the conflict messages in the code. Not sure how this skipped CI but these all have to be removed.